src/app/config.rs

```diff
@@ -90,6 +94,12 @@ pub struct Config {
 
     /// Allow uptime checks against internal IP addresses
     pub allow_internal_ips: bool,
+
+    /// Name of the pod that is running
+    pub checker_id: String,
```

Comment:
Left this comment somewherer else, but wondiering if we should do the id deserialization here.

Could deserialize into a struct like `CheckerId` that includes the numeric value

Reply:
I'll do this in https://github.com/getsentry/uptime-checker/pull/214